aclk/aclk_stats.c

```diff
@@ -110,6 +110,86 @@ static void aclk_stats_cloud_req(struct aclk_metrics_per_sample *per_sample)
     rrdset_done(st);
 }
 
+static void aclk_stats_cloud_req_type(struct aclk_metrics_per_sample *per_sample)
+{
+    static RRDSET *st = NULL;
+    static RRDDIM *rd_type_http = NULL;
+    static RRDDIM *rd_type_alarm_upd = NULL;
+    static RRDDIM *rd_type_metadata_info = NULL;
+    static RRDDIM *rd_type_metadata_alarms = NULL;
+    static RRDDIM *rd_type_chart_new = NULL;
+    static RRDDIM *rd_type_chart_del = NULL;
+    static RRDDIM *rd_type_register_node = NULL;
+    static RRDDIM *rd_type_node_upd = NULL;
+
+    if (unlikely(!st)) {
+        st = rrdset_create_localhost(
+            "netdata", "aclk_cloud_req_type", NULL, "aclk", NULL, "Requests received from cloud via their type", "req/s",
```

Comment:
i think copy paste typo.... `via` should be `by`? (it makes sense in the other chart)

Reply:
In this case it wasn't actually a typo, because I think `via` can *technically* make some sense here, but definitely `by` is much better - thanks!